k2/python/k2/symbol_table.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+#
+# See ../../../LICENSE for clarification regarding multiple authors
+
+from typing import Dict
+from typing import Union
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class SymbolTable(object):
+    _id2sym: Dict[int, str]
+    '''Map an integer to a symbol.
+    '''
+
+    _sym2id: Dict[str, int]
+    '''Map a symbol to an integer.
+    '''
+
```

Comment:
We may want to add a function for validation, and check if it's a valid symbol table. For example:
* If 0 is used for other symbols
* If we have <eps>, and if we don't, add that

Then when we initialize this symbol table, we should run the validation.

Reply:
Thanks, fixed.